lib/init.coffee

```diff
@@ -10,6 +15,82 @@ module.exports =
       type: 'string'
       default: ''
 
-
   activate: ->
-    console.log 'activate linter-erlc'
+    @subscriptions = new CompositeDisposable
+    @subscriptions.add atom.config.observe 'linter-erlc.executablePath',
+    (executablePath) =>
+      @executablePath = executablePath
+  deactivate: ->
+    @subscriptions.dispose()
+  provideLinter: ->
+    helpers = require('atom-linter')
+    provider =
+      name: 'Erlc'
+      grammarScopes: ['source.erlang']
+      scope: 'file'
+      lintOnFly: false
+      lint: (textEditor) =>
+        filePath = textEditor.getPath()
+        args = ["-Wall", "-v", "-o", os.tmpDir()]
```

Comment:
Looks like the answer is to include [rimraf](https://www.npmjs.com/package/rimraf) and use it to delete this temporary directory.